File: cfgs/java_cfgs_foo/PROGRAM_DECIMAL_BINARY_CONVERSION_2.java
Nodes
1;special;;;
2;statement;int B_Number = 0;14;
3;statement;int cnt = 0;15;
4;conditional;foo_1 != 0;16;
5;statement;int rem = foo_1 % 2;17;
6;statement;double c = fun_1(10, foo_1);18;
7;statement;B_Number += foo_1 * foo_2;19;
8;statement;N /= 2;20;
9;statement;cnt++;21;
10;statement;return foo_1;23;
11;exit;;;
Edges
0;2;
1;3;
2;4;
3;10;5;
4;6;
5;7;
6;8;
7;9;
8;4;
9;11;
10;

File: cfgs/pythoncfgs/PROGRAM_DECIMAL_BINARY_CONVERSION_2.py
Nodes
1;statement;foo_1 = 0;8;
2;statement;foo_1 = 0;9;
3;conditional;foo_1 != 0;10;
4;statement;return foo_1;16;
5;statement;foo_1 = foo_2 % 2;11;
6;statement;foo_1 = fun_1(10, foo_2);12;
7;statement;foo_1 += foo_2 * foo_3;13;
8;statement;foo_1 //= 2;14;
9;statement;foo_1 += 1;15;
10;exit;;
Edges
0;2;
1;3;
2;4;5;
3;10;
4;6;
5;7;
6;8;
7;9;
8;3;
9;
